source/chapters/getting_started.rst

```diff
@@ -74,19 +74,31 @@ Analyze your library
    S.Brandt <s.brandt@mixxx.org>
 
 |ic_lib_prepare| Mixxx automatically analyzes tracks the first time you load
-them in a deck, nevertheless it is recommended that you analyze them before
-playing live to ensure the beatgrids are correct. Furthermore, track
+them in a deck. This allows you to run :term:`beatgrid`, :term:`key`, and :term:`ReplayGain` detection on tracks in advance. It also prepares the waveform overviews, detects file corruption, and calculates the BPM values.
+
+.. seealso:: For more information, go to :ref:`configuration-bpm-detection`.
+
+Why you should analyze your audio files
+---------------------------------------
+
+If the tracks have never been played with Mixxx before, the library columns :guilabel:`BPM`,
+:guilabel:`Duration` and :guilabel:`Key` will be empty.
+It is recommended that you analyze your audio files before
+playing live to ensure the beatgrids are correct and these fields show. Furthermore, track
```

Comment:
we could also say something like  "pregenerating beatgrids gives you the chance to doublecheck the beat markers to make sure they are correct.  Errors in beatgrids can cause difficulty when syncing tracks"

Reply:
Alright